src/components/history/index.tsx

```diff
@@ -1,15 +1,52 @@
 import React from 'react';
-import { Box, makeStyles } from '@material-ui/core';
+import {
+    Avatar,
+    Box,
+    List, 
+    ListItem,
+    ListItemAvatar,
+    ListItemText,
+} from '@material-ui/core';
+import HowToRegIcon from '@material-ui/icons/HowToReg';
+import VpnKeyIcon from '@material-ui/icons/VpnKey';
 
-const useStyles = makeStyles(() => ({
-    // TODO
-}));
+type History = {
+    type: string,
+    createdAt: string,
+    origin: string,
+};
+
+type HistoryListPageProps = {
+    histories: Array<History>
+};
 
-export default function HistoryListPage () {
+export default function HistoryListPage ({ histories }: HistoryListPageProps){
     
     return (
         <Box>
-            history
+            <List>
+                {
+                    histories.map((history) => {
+                        return (
+                            <ListItem divider>
+                                <ListItemAvatar>
+                                    <Avatar>
+                                        {
+                                            history.type === 'REGISTRATION'
+                                                ? <HowToRegIcon />
+                                                : <VpnKeyIcon />
+                                        }
+                                    </Avatar>
+                                </ListItemAvatar>
+                                <ListItemText 
+                                    primary={history.type} 
+                                    secondary={`${new Date(history.createdAt).toDateString()}  ${history.origin}`} 
+                                />
+                            </ListItem>
+                        )
+                    })
+                }
+            </List>
```

Comment:
Appearance.
<img width="349" alt="스크린샷 2020-09-03 오전 2 30 22" src="https://user-images.githubusercontent.com/14817195/92016535-6cd08d00-ed8d-11ea-8b1d-3a26105d07e7.png">

Reply:
Looks good enough in my opinion.